### DISCUSSION TITLE
Customize rebase checkbox text

### DISCUSSION THREAD

**Discussion by Atches** [renovatebot/renovate#24866] [Suggest an Idea]
### What would you like Renovate to be able to do?

Provide a new configuration option for customizing the rebase checkbox text.

### If you have any ideas on how this should be implemented, please tell us here.

Allow support in this function: 
https://github.com/renovatebot/renovate/blob/b30ac1a4c2a331b9331fbe175ac8a97547367695/lib/workers/repository/update/pr/body/controls.ts#L1-L3

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by rarkins** [comment]
Although this is written as a function, it seems we haven't had any need to utilize that until now. Potentially we could convert this to a string `prControls` and make the configurable either by global or repo config.
